### ISSUE TITLE
Build from master fails without GOPROXY set

### ISSUE THREAD

**Issue by jinnko** [restic/restic#2851]
Output of `restic version`
--------------------------

Building from master

How did you run restic exactly?
-------------------------------

Built restic/builder locally, then followed instructions to build from master.

What backend/server/service did you use to store the repository?
----------------------------------------------------------------

N/A

Expected behavior
-----------------

Build should work as described in docs.

Actual behavior
---------------

Build fails as follows:

```
# docker run --rm --volume "$PWD/restic:/restic" --volume "$PWD/output:/output" restic/builder
build github.com/spf13/pflag: cannot load github.com/spf13/pflag: open /restic/vendor/github.com/spf13/pflag: no such file or directory
```

```
# docker run --rm --volume "$PWD/restic:/restic" --volume "$PWD/output:/output" restic/builder go run helpers/build-release-binaries/main.go --verbose
go: bazil.org/fuse@v0.0.0-20191225072544-27e78e7d88df: module lookup disabled by GOPROXY=off
```

```
# docker run -e GOPROXY=https://proxy.golang.org,direct --rm --volume "$PWD/restic:/restic" --volume "$PWD/output:/output" restic/builder go run helpers/build-release-binaries/main.go --verbose
go: downloading golang.org/x/sync v0.0.0-20190423024810-112230192c58
go: downloading github.com/spf13/pflag v1.0.3
go: extracting golang.org/x/sync v0.0.0-20190423024810-112230192c58
go: extracting github.com/spf13/pflag v1.0.3
go: finding github.com/spf13/pflag v1.0.3
go: finding golang.org/x/sync v0.0.0-20190423024810-112230192c58
building with 16 workers
build linux/amd64
build linux/arm
build darwin/amd64
run go [go build -o /output/restic_linux_amd64 -ldflags -s -w -tags selfupdate ./cmd/restic] in /restic
build freebsd/386
run go [go build -o /output/restic_linux_arm -ldflags -s -w -tags selfupdate ./cmd/restic] in /restic
build windows/386
build windows/amd64
run go [go build -o /output/restic_freebsd_386 -ldflags -s -w -tags selfupdate ./cmd/restic] in /restic
run go [go build -o /output/restic_windows_386.exe -ldflags -s -w -tags selfupdate ./cmd/restic] in /restic
run go [go build -o /output/restic_windows_amd64.exe -ldflags -s -w -tags selfupdate ./cmd/restic] in /restic
run go [go build -o /output/restic_darwin_amd64 -ldflags -s -w -tags selfupdate ./cmd/restic] in /restic
build linux/386
build freebsd/arm
build openbsd/386
build openbsd/amd64
build linux/ppc64le
build linux/arm64
run go [go build -o /output/restic_openbsd_386 -ldflags -s -w -tags selfupdate ./cmd/restic] in /restic
run go [go build -o /output/restic_linux_arm64 -ldflags -s -w -tags selfupdate ./cmd/restic] in /restic
run go [go build -o /output/restic_freebsd_arm -ldflags -s -w -tags selfupdate ./cmd/restic] in /restic
build darwin/386
run go [go build -o /output/restic_linux_ppc64le -ldflags -s -w -tags selfupdate ./cmd/restic] in /restic
build freebsd/amd64
run go [go build -o /output/restic_darwin_386 -ldflags -s -w -tags selfupdate ./cmd/restic] in /restic
run go [go build -o /output/restic_openbsd_amd64 -ldflags -s -w -tags selfupdate ./cmd/restic] in /restic
run go [go build -o /output/restic_freebsd_amd64 -ldflags -s -w -tags selfupdate ./cmd/restic] in /restic
run go [go build -o /output/restic_linux_386 -ldflags -s -w -tags selfupdate ./cmd/restic] in /restic
go: downloading golang.org/x/crypto v0.0.0-20200427165652-729f1e841bcc
go: downloading bazil.org/fuse v0.0.0-20191225072544-27e78e7d88df
go: downloading golang.org/x/oauth2 v0.0.0-20190402181905-9f3314589c9a
go: downloading google.golang.org/api v0.3.2
go: downloading github.com/minio/sha256-simd v0.1.1
go: downloading gopkg.in/tomb.v2 v2.0.0-20161208151619-d5d1b5820637
go: downloading github.com/spf13/cobra v0.0.3
go: downloading github.com/ncw/swift v1.0.47
go: downloading github.com/dchest/siphash v1.2.1
go: downloading bazil.org/fuse v0.0.0-20191225072544-27e78e7d88df
go: downloading golang.org/x/crypto v0.0.0-20200427165652-729f1e841bcc
go: downloading github.com/Azure/azure-sdk-for-go v27.3.0+incompatible
go: downloading github.com/pkg/xattr v0.4.1
go: downloading golang.org/x/net v0.0.0-20200425230154-ff2c4b7c35a0
go: downloading github.com/juju/ratelimit v1.0.1
go: downloading golang.org/x/oauth2 v0.0.0-20190402181905-9f3314589c9a
go: downloading bazil.org/fuse v0.0.0-20191225072544-27e78e7d88df
go: downloading gopkg.in/tomb.v2 v2.0.0-20161208151619-d5d1b5820637
go: downloading github.com/minio/sha256-simd v0.1.1
go: downloading github.com/cespare/xxhash v1.1.0
go: downloading github.com/juju/ratelimit v1.0.1
go: downloading github.com/juju/ratelimit v1.0.1
go: downloading github.com/pkg/sftp v1.10.0
go: downloading golang.org/x/net v0.0.0-20200425230154-ff2c4b7c35a0
go: downloading github.com/pkg/sftp v1.10.0
go: downloading github.com/spf13/cobra v0.0.3
go: downloading github.com/spf13/cobra v0.0.3
go: downloading github.com/minio/minio-go/v6 v6.0.43
go: downloading github.com/restic/chunker v0.4.0
go: downloading github.com/minio/sha256-simd v0.1.1
go: downloading github.com/minio/minio-go/v6 v6.0.43
go: downloading golang.org/x/net v0.0.0-20200425230154-ff2c4b7c35a0
go: downloading github.com/pkg/errors v0.8.1
go: downloading github.com/spf13/cobra v0.0.3
go: downloading golang.org/x/crypto v0.0.0-20200427165652-729f1e841bcc
go: downloading gopkg.in/tomb.v2 v2.0.0-20161208151619-d5d1b5820637
go: downloading github.com/Azure/azure-sdk-for-go v27.3.0+incompatible
go: downloading golang.org/x/text v0.3.1-0.20180807135948-17ff2d5776d2
go: downloading github.com/minio/minio-go/v6 v6.0.43
go: downloading google.golang.org/api v0.3.2
go: downloading github.com/kurin/blazer v0.5.3
go: downloading github.com/cenkalti/backoff v2.1.1+incompatible
go: downloading github.com/cenkalti/backoff v2.1.1+incompatible
go: downloading google.golang.org/api v0.3.2
go: downloading github.com/juju/ratelimit v1.0.1
go: downloading golang.org/x/crypto v0.0.0-20200427165652-729f1e841bcc
go: downloading github.com/elithrar/simple-scrypt v1.3.0
go: downloading golang.org/x/sys v0.0.0-20200323222414-85ca7c5b95cd
go: downloading github.com/minio/sha256-simd v0.1.1
go: downloading golang.org/x/crypto v0.0.0-20200427165652-729f1e841bcc
go: downloading golang.org/x/crypto v0.0.0-20200427165652-729f1e841bcc
go: downloading github.com/pkg/errors v0.8.1
go: downloading github.com/spf13/cobra v0.0.3
go: downloading github.com/Azure/azure-sdk-for-go v27.3.0+incompatible
go: downloading github.com/dchest/siphash v1.2.1
go: downloading github.com/pkg/errors v0.8.1
go: downloading github.com/minio/minio-go/v6 v6.0.43
go: downloading gopkg.in/tomb.v2 v2.0.0-20161208151619-d5d1b5820637
go: downloading github.com/pkg/xattr v0.4.1
go: downloading gopkg.in/tomb.v2 v2.0.0-20161208151619-d5d1b5820637
go: downloading github.com/cespare/xxhash v1.1.0
go: downloading bazil.org/fuse v0.0.0-20191225072544-27e78e7d88df
go: downloading github.com/spf13/cobra v0.0.3
go: downloading github.com/hashicorp/golang-lru v0.5.1
go: downloading golang.org/x/net v0.0.0-20200425230154-ff2c4b7c35a0
go: downloading golang.org/x/crypto v0.0.0-20200427165652-729f1e841bcc
go: downloading github.com/pkg/sftp v1.10.0
go: downloading github.com/kurin/blazer v0.5.3
go: downloading github.com/dchest/siphash v1.2.1
go: downloading github.com/juju/ratelimit v1.0.1
go: downloading github.com/ncw/swift v1.0.47
go: downloading github.com/minio/minio-go/v6 v6.0.43
go: downloading github.com/pkg/errors v0.8.1
go: downloading github.com/spf13/cobra v0.0.3
go: downloading github.com/cespare/xxhash v1.1.0
go: downloading github.com/dchest/siphash v1.2.1
go: downloading golang.org/x/text v0.3.1-0.20180807135948-17ff2d5776d2
go: downloading google.golang.org/api v0.3.2
go: downloading github.com/elithrar/simple-scrypt v1.3.0
go: downloading github.com/cenkalti/backoff v2.1.1+incompatible
go: downloading github.com/minio/sha256-simd v0.1.1
go: downloading github.com/juju/ratelimit v1.0.1
go: downloading github.com/spf13/cobra v0.0.3
go: downloading github.com/pkg/sftp v1.10.0
go: downloading github.com/Azure/azure-sdk-for-go v27.3.0+incompatible
go: downloading bazil.org/fuse v0.0.0-20191225072544-27e78e7d88df
go: downloading golang.org/x/net v0.0.0-20200425230154-ff2c4b7c35a0
go: downloading github.com/Azure/azure-sdk-for-go v27.3.0+incompatible
go: downloading github.com/cenkalti/backoff v2.1.1+incompatible
go: downloading gopkg.in/tomb.v2 v2.0.0-20161208151619-d5d1b5820637
go: downloading golang.org/x/oauth2 v0.0.0-20190402181905-9f3314589c9a
go: downloading github.com/spf13/cobra v0.0.3
go: downloading golang.org/x/oauth2 v0.0.0-20190402181905-9f3314589c9a
go: downloading golang.org/x/crypto v0.0.0-20200427165652-729f1e841bcc
go: downloading golang.org/x/net v0.0.0-20200425230154-ff2c4b7c35a0
go: downloading google.golang.org/api v0.3.2
go: downloading github.com/minio/minio-go/v6 v6.0.43
go: downloading github.com/cenkalti/backoff v2.1.1+incompatible
go: downloading github.com/cenkalti/backoff v2.1.1+incompatible
go: downloading github.com/pkg/xattr v0.4.1
go: downloading golang.org/x/net v0.0.0-20200425230154-ff2c4b7c35a0
go: downloading golang.org/x/crypto v0.0.0-20200427165652-729f1e841bcc
go: downloading github.com/elithrar/simple-scrypt v1.3.0
go: downloading github.com/hashicorp/golang-lru v0.5.1
go: downloading github.com/elithrar/simple-scrypt v1.3.0
go: downloading gopkg.in/tomb.v2 v2.0.0-20161208151619-d5d1b5820637
go: downloading github.com/juju/ratelimit v1.0.1
go: downloading golang.org/x/net v0.0.0-20200425230154-ff2c4b7c35a0
go: downloading github.com/minio/minio-go/v6 v6.0.43
go: downloading golang.org/x/sys v0.0.0-20200323222414-85ca7c5b95cd
go: downloading github.com/dchest/siphash v1.2.1
go: downloading github.com/Azure/azure-sdk-for-go v27.3.0+incompatible
go: downloading gopkg.in/tomb.v2 v2.0.0-20161208151619-d5d1b5820637
go: downloading github.com/elithrar/simple-scrypt v1.3.0
go: downloading github.com/pkg/errors v0.8.1
go: downloading gopkg.in/tomb.v2 v2.0.0-20161208151619-d5d1b5820637
go: downloading github.com/restic/chunker v0.4.0
go: downloading github.com/pkg/errors v0.8.1
go: downloading github.com/hashicorp/golang-lru v0.5.1
go: downloading github.com/restic/chunker v0.4.0
go: downloading github.com/cespare/xxhash v1.1.0
go: downloading golang.org/x/text v0.3.1-0.20180807135948-17ff2d5776d2
go: downloading golang.org/x/oauth2 v0.0.0-20190402181905-9f3314589c9a
go: downloading github.com/kurin/blazer v0.5.3
go: downloading github.com/pkg/sftp v1.10.0
go: downloading github.com/spf13/cobra v0.0.3
go: downloading github.com/pkg/errors v0.8.1
go: downloading golang.org/x/crypto v0.0.0-20200427165652-729f1e841bcc
go: downloading golang.org/x/oauth2 v0.0.0-20190402181905-9f3314589c9a
go: downloading github.com/elithrar/simple-scrypt v1.3.0
go: downloading github.com/restic/chunker v0.4.0
go: downloading bazil.org/fuse v0.0.0-20191225072544-27e78e7d88df
go: downloading golang.org/x/net v0.0.0-20200425230154-ff2c4b7c35a0
go: downloading github.com/ncw/swift v1.0.47
go: downloading github.com/restic/chunker v0.4.0
go: downloading github.com/minio/minio-go/v6 v6.0.43
go: downloading github.com/cespare/xxhash v1.1.0
go: downloading golang.org/x/sys v0.0.0-20200323222414-85ca7c5b95cd
go: downloading github.com/pkg/errors v0.8.1
go: downloading golang.org/x/net v0.0.0-20200425230154-ff2c4b7c35a0
go: downloading github.com/hashicorp/golang-lru v0.5.1
go: downloading github.com/kurin/blazer v0.5.3
go: downloading github.com/minio/sha256-simd v0.1.1
go: downloading github.com/juju/ratelimit v1.0.1
go: downloading github.com/cespare/xxhash v1.1.0
go: downloading github.com/ncw/swift v1.0.47
go: downloading golang.org/x/text v0.3.1-0.20180807135948-17ff2d5776d2
go: downloading github.com/dchest/siphash v1.2.1
go: downloading google.golang.org/api v0.3.2
go: downloading github.com/ncw/swift v1.0.47
go: downloading golang.org/x/text v0.3.1-0.20180807135948-17ff2d5776d2
go: downloading github.com/cenkalti/backoff v2.1.1+incompatible
go: downloading github.com/pkg/sftp v1.10.0
go: downloading github.com/ncw/swift v1.0.47
go: downloading github.com/pkg/xattr v0.4.1
go: downloading github.com/restic/chunker v0.4.0
go: downloading github.com/cespare/xxhash v1.1.0
go: downloading github.com/kurin/blazer v0.5.3
go: downloading github.com/minio/minio-go/v6 v6.0.43
go: downloading github.com/Azure/azure-sdk-for-go v27.3.0+incompatible
go: downloading golang.org/x/text v0.3.1-0.20180807135948-17ff2d5776d2
go: downloading github.com/cenkalti/backoff v2.1.1+incompatible
go: downloading bazil.org/fuse v0.0.0-20191225072544-27e78e7d88df
go: downloading google.golang.org/api v0.3.2
go: downloading github.com/pkg/sftp v1.10.0
go: downloading github.com/elithrar/simple-scrypt v1.3.0
go: downloading github.com/ncw/swift v1.0.47
go: downloading github.com/ncw/swift v1.0.47
go: downloading github.com/pkg/errors v0.8.1
go: downloading github.com/minio/sha256-simd v0.1.1
go: downloading github.com/pkg/errors v0.8.1
go: downloading github.com/hashicorp/golang-lru v0.5.1
go: downloading github.com/pkg/xattr v0.4.1
go: downloading github.com/ncw/swift v1.0.47
go: downloading github.com/spf13/cobra v0.0.3
go: downloading github.com/minio/sha256-simd v0.1.1
go: downloading golang.org/x/text v0.3.1-0.20180807135948-17ff2d5776d2
go: downloading golang.org/x/oauth2 v0.0.0-20190402181905-9f3314589c9a
go: downloading github.com/ncw/swift v1.0.47
go: downloading golang.org/x/oauth2 v0.0.0-20190402181905-9f3314589c9a
go: downloading github.com/pkg/sftp v1.10.0
go: downloading google.golang.org/api v0.3.2
go: downloading github.com/hashicorp/golang-lru v0.5.1
go: downloading golang.org/x/text v0.3.1-0.20180807135948-17ff2d5776d2
go: downloading github.com/cespare/xxhash v1.1.0
go: downloading github.com/kurin/blazer v0.5.3
go: downloading github.com/restic/chunker v0.4.0
go: downloading github.com/dchest/siphash v1.2.1
go: downloading golang.org/x/sys v0.0.0-20200323222414-85ca7c5b95cd
go: downloading github.com/Azure/azure-sdk-for-go v27.3.0+incompatible
go: downloading github.com/pkg/sftp v1.10.0
go: downloading github.com/elithrar/simple-scrypt v1.3.0
go: downloading github.com/cespare/xxhash v1.1.0
go: downloading github.com/pkg/xattr v0.4.1
go: downloading github.com/minio/sha256-simd v0.1.1
go: downloading github.com/kurin/blazer v0.5.3
go: downloading github.com/restic/chunker v0.4.0
go: downloading golang.org/x/crypto v0.0.0-20200427165652-729f1e841bcc
go: downloading github.com/minio/minio-go/v6 v6.0.43
go: downloading github.com/minio/sha256-simd v0.1.1
go: downloading github.com/kurin/blazer v0.5.3
go: downloading golang.org/x/text v0.3.1-0.20180807135948-17ff2d5776d2
go: downloading github.com/pkg/xattr v0.4.1
go: downloading golang.org/x/net v0.0.0-20200425230154-ff2c4b7c35a0
go: downloading github.com/restic/chunker v0.4.0
go: downloading github.com/Azure/azure-sdk-for-go v27.3.0+incompatible
go: downloading golang.org/x/text v0.3.1-0.20180807135948-17ff2d5776d2
go: downloading github.com/cespare/xxhash v1.1.0
go: downloading github.com/juju/ratelimit v1.0.1
go: downloading github.com/dchest/siphash v1.2.1
go: downloading gopkg.in/tomb.v2 v2.0.0-20161208151619-d5d1b5820637
go: downloading golang.org/x/text v0.3.1-0.20180807135948-17ff2d5776d2
go: downloading golang.org/x/oauth2 v0.0.0-20190402181905-9f3314589c9a
go: downloading github.com/dchest/siphash v1.2.1
go: downloading google.golang.org/api v0.3.2
go: downloading github.com/hashicorp/golang-lru v0.5.1
go: downloading github.com/spf13/cobra v0.0.3
go: downloading github.com/ncw/swift v1.0.47
go: downloading golang.org/x/oauth2 v0.0.0-20190402181905-9f3314589c9a
go: downloading golang.org/x/sys v0.0.0-20200323222414-85ca7c5b95cd
go: downloading github.com/cenkalti/backoff v2.1.1+incompatible
go: downloading github.com/minio/sha256-simd v0.1.1
go: downloading github.com/Azure/azure-sdk-for-go v27.3.0+incompatible
go: downloading github.com/kurin/blazer v0.5.3
go: downloading github.com/juju/ratelimit v1.0.1
go: downloading golang.org/x/net v0.0.0-20200425230154-ff2c4b7c35a0
go: downloading google.golang.org/api v0.3.2
go: downloading gopkg.in/tomb.v2 v2.0.0-20161208151619-d5d1b5820637
go: downloading github.com/restic/chunker v0.4.0
go: downloading golang.org/x/crypto v0.0.0-20200427165652-729f1e841bcc
go: downloading github.com/pkg/sftp v1.10.0
go: downloading github.com/elithrar/simple-scrypt v1.3.0
go: downloading github.com/minio/sha256-simd v0.1.1
go: downloading github.com/cenkalti/backoff v2.1.1+incompatible
go: downloading github.com/Azure/azure-sdk-for-go v27.3.0+incompatible
go: downloading github.com/kurin/blazer v0.5.3
go: downloading gopkg.in/tomb.v2 v2.0.0-20161208151619-d5d1b5820637
go: downloading golang.org/x/oauth2 v0.0.0-20190402181905-9f3314589c9a
go: downloading golang.org/x/oauth2 v0.0.0-20190402181905-9f3314589c9a
go: downloading github.com/Azure/azure-sdk-for-go v27.3.0+incompatible
go: downloading github.com/Azure/azure-sdk-for-go v27.3.0+incompatible
go: downloading google.golang.org/api v0.3.2
go: downloading github.com/dchest/siphash v1.2.1
go: downloading bazil.org/fuse v0.0.0-20191225072544-27e78e7d88df
go: downloading gopkg.in/tomb.v2 v2.0.0-20161208151619-d5d1b5820637
go: downloading golang.org/x/text v0.3.1-0.20180807135948-17ff2d5776d2
go: downloading github.com/minio/sha256-simd v0.1.1
go: downloading github.com/hashicorp/golang-lru v0.5.1
go: downloading bazil.org/fuse v0.0.0-20191225072544-27e78e7d88df
go: downloading github.com/pkg/sftp v1.10.0
go: downloading github.com/pkg/errors v0.8.1
go: downloading github.com/elithrar/simple-scrypt v1.3.0
go: downloading golang.org/x/crypto v0.0.0-20200427165652-729f1e841bcc
go: downloading github.com/restic/chunker v0.4.0
go: downloading github.com/ncw/swift v1.0.47
go: downloading github.com/pkg/xattr v0.4.1
go: downloading github.com/hashicorp/golang-lru v0.5.1
go: downloading golang.org/x/crypto v0.0.0-20200427165652-729f1e841bcc
go: downloading github.com/dchest/siphash v1.2.1
go: downloading github.com/kurin/blazer v0.5.3
go: downloading bazil.org/fuse v0.0.0-20191225072544-27e78e7d88df
go: downloading github.com/restic/chunker v0.4.0
go: downloading github.com/restic/chunker v0.4.0
go: downloading github.com/cenkalti/backoff v2.1.1+incompatible
go: downloading golang.org/x/text v0.3.1-0.20180807135948-17ff2d5776d2
go: downloading github.com/dchest/siphash v1.2.1
go: downloading google.golang.org/api v0.3.2
go: downloading github.com/minio/minio-go/v6 v6.0.43
go: downloading github.com/pkg/sftp v1.10.0
go: downloading golang.org/x/oauth2 v0.0.0-20190402181905-9f3314589c9a
go: downloading github.com/cespare/xxhash v1.1.0
go: downloading github.com/spf13/cobra v0.0.3
go: downloading gopkg.in/tomb.v2 v2.0.0-20161208151619-d5d1b5820637
go: downloading github.com/elithrar/simple-scrypt v1.3.0
go: downloading github.com/cenkalti/backoff v2.1.1+incompatible
go: downloading github.com/juju/ratelimit v1.0.1
go: downloading github.com/elithrar/simple-scrypt v1.3.0
go: downloading github.com/pkg/errors v0.8.1
go: downloading github.com/minio/minio-go/v6 v6.0.43
go: downloading golang.org/x/net v0.0.0-20200425230154-ff2c4b7c35a0
go: downloading github.com/dchest/siphash v1.2.1
go: downloading github.com/spf13/cobra v0.0.3
go: downloading github.com/Azure/azure-sdk-for-go v27.3.0+incompatible
go: downloading github.com/elithrar/simple-scrypt v1.3.0
go: downloading github.com/cenkalti/backoff v2.1.1+incompatible
go: downloading github.com/kurin/blazer v0.5.3
go: downloading golang.org/x/net v0.0.0-20200425230154-ff2c4b7c35a0
go: downloading github.com/ncw/swift v1.0.47
go: downloading github.com/pkg/errors v0.8.1
go: downloading github.com/cespare/xxhash v1.1.0
go: downloading golang.org/x/text v0.3.1-0.20180807135948-17ff2d5776d2
go: downloading github.com/juju/ratelimit v1.0.1
go: downloading github.com/restic/chunker v0.4.0
go: downloading github.com/minio/minio-go/v6 v6.0.43
go: downloading github.com/cespare/xxhash v1.1.0
go: downloading github.com/kurin/blazer v0.5.3
go: downloading github.com/juju/ratelimit v1.0.1
go: downloading github.com/pkg/xattr v0.4.1
go: downloading github.com/cenkalti/backoff v2.1.1+incompatible
go: downloading google.golang.org/api v0.3.2
go: downloading github.com/elithrar/simple-scrypt v1.3.0
go: downloading github.com/minio/sha256-simd v0.1.1
go: downloading github.com/hashicorp/golang-lru v0.5.1
go: downloading github.com/minio/minio-go/v6 v6.0.43
go: downloading golang.org/x/sys v0.0.0-20200323222414-85ca7c5b95cd
go: downloading github.com/dchest/siphash v1.2.1
go: downloading github.com/cespare/xxhash v1.1.0
go: downloading github.com/juju/ratelimit v1.0.1
go: downloading github.com/pkg/errors v0.8.1
go: downloading golang.org/x/oauth2 v0.0.0-20190402181905-9f3314589c9a
go: downloading github.com/pkg/sftp v1.10.0
go: downloading github.com/ncw/swift v1.0.47
go: downloading github.com/kurin/blazer v0.5.3
go: downloading google.golang.org/api v0.3.2
go: downloading golang.org/x/sys v0.0.0-20200323222414-85ca7c5b95cd
go: downloading github.com/pkg/xattr v0.4.1
go: downloading github.com/pkg/sftp v1.10.0
build github.com/restic/restic/cmd/restic: cannot load bazil.org/fuse: module lookup disabled by GOPROXY=off
build github.com/restic/restic/cmd/restic: cannot load bazil.org/fuse: module lookup disabled by GOPROXY=off
error building linux/arm: exit status 1
build github.com/restic/restic/cmd/restic: cannot load bazil.org/fuse: module lookup disabled by GOPROXY=off
exit status 1
build github.com/restic/restic/cmd/restic: cannot load bazil.org/fuse: module lookup disabled by GOPROXY=off
build github.com/restic/restic/cmd/restic: cannot load github.com/Azure/azure-sdk-for-go/storage: module lookup disabled by GOPROXY=off
build github.com/restic/restic/cmd/restic: cannot load bazil.org/fuse: module lookup disabled by GOPROXY=off
build github.com/restic/restic/cmd/restic: cannot load github.com/Azure/azure-sdk-for-go/storage: module lookup disabled by GOPROXY=off
build github.com/restic/restic/cmd/restic: cannot load bazil.org/fuse: module lookup disabled by GOPROXY=off
build github.com/restic/restic/cmd/restic: cannot load github.com/Azure/azure-sdk-for-go/storage: module lookup disabled by GOPROXY=off
build github.com/restic/restic/cmd/restic: cannot load bazil.org/fuse: module lookup disabled by GOPROXY=off
build github.com/restic/restic/cmd/restic: cannot load github.com/Azure/azure-sdk-for-go/storage: module lookup disabled by GOPROXY=off
build github.com/restic/restic/cmd/restic: cannot load bazil.org/fuse: module lookup disabled by GOPROXY=off
build github.com/restic/restic/cmd/restic: cannot load bazil.org/fuse: module lookup disabled by GOPROXY=off
build github.com/restic/restic/cmd/restic: cannot load bazil.org/fuse: module lookup disabled by GOPROXY=off
```

Steps to reproduce the behavior
-------------------------------

1. Create local version of restic/builder as described in docs
2. Get latest restic source
3. Build restic from master as outlined above

Do you have any idea what may have caused this?
-----------------------------------------------

Do you have an idea how to solve the issue?
-------------------------------------------

1. Edit `restic/helpers/build-release-binaries/main.go` and set `GOPROXY=https://proxy.golang.org` on line 113.
2. Set `-e GOPROXY=https://proxy.golang.org,direct` when running the container.
3. Build completes successfully

I also attempted a golang update first by building restic/builder with golang-1.13.14 hoping that might help, but it didn't.  So the final build was actually created with the newer golang.

Did restic help you today? Did it make you happy in any way?
------------------------------------------------------------

I really like restic.  Have been using it on multiple systems against a central rest server for a while and its simplicity is great, along with deduplication.  Today I'm trying to get backups working on a system with 2GB RAM over SFTP which is failing with out of memory errors, hence the attempt to build from source as I saw some comments in tickets from the last couple of weeks suggesting there's fixes on master.  Fingers crossed this master build will help.

**Comment by aawsome** [comment]
You can try the [beta releases](https://beta.restic.net/?sort=time&order=asc)

**Comment by fd0** [comment]
Hm, I'm not sure where you found the instructions. The Docker container is just used to build the official release binaries, it's not meant to be run by users :) 

The [instructions](https://restic.readthedocs.io/en/latest/020_installation.html#from-source) in the manual tell users to use `go run build.go`, which works for releases as well as the master branch.

Is there anything we could improve?

**Comment by dimejo** [comment]
Looks like @jinnko was using the documentation for [Building the Official Binaries](https://restic.readthedocs.io/en/latest/developer_information.html#building-the-official-binaries).

**Comment by jinnko** [comment]
Indeed I used the Building the Official Binaries you linked.  When I saw the docker instructions I thought it was a clean, organized and deterministic way of building.  My bad for missing the Install from Source section, and I don't think there's really much to be improved in the docs.

Unfortunately the master build didn't solve my issue.  I'm backing up about 2 GB of data, but unfortunately there's around 380,000 files in a single folder (Grafana Loki data store).  It seems the data is transferred to the repo and restic fails after that as a `check` finds lots of orphaned objects.  I appreciate it's not a great situation to be in, but I'm happy to raise a separate ticket if you think it's an issue worth investigating.
